libraries/AP_RCProtocol/AP_RCProtocol_SUMD.cpp

```diff
@@ -169,23 +169,26 @@ void AP_RCProtocol_SUMD::_process_byte(uint32_t timestamp_us, uint8_t byte)
             /* reorder first 4 channels */
 
             /* ch1 = roll -> sumd = ch2 */
-            values[0] = (uint16_t)((_rxpacket.sumd_data[1 * 2 + 1] << 8) | _rxpacket.sumd_data[1 * 2 + 2]) >> 3;
```

Comment:
I must be blind as I don't see the bug here, it seems to be dividing the overall uint16_t value by 8, not just 1 byte? what am I missing?